src/util/apiUtils.js

```diff
@@ -270,10 +275,13 @@ export function parseRequestHeaders(request) {
 export function parseRequestQueries(request) {
 	const {
 		method,
+		mime,
```

Comment:
For better or worse, you have to forget everything you know about Redux when you're working in server-side request handling code. The `request` here is a [Hapi Request Object](https://hapijs.com/api#request-object), which has a `mime` property derived from the `Content-Type` header of the request - it's not populated manually.